openid-4-verifiable-presentations-1_0.md

```diff
@@ -1749,6 +1771,8 @@ Setting `limit_disclosure` property defined in [@!DIF.PresentationExchange] to `
 
 A non-normative example of the Authorization Response would look the same as in the examples of other Credential formats in this Annex.
 
+`transaction_data` response parameter defined in (#transaction-data) MUST be included in the Key Binding JWT as a top level parameter.
```

Comment:
The link is invalid, I believe `-` should be `_`: 
```suggestion
`transaction_data` response parameter defined in (#transaction_data) MUST be included in the Key Binding JWT as a top level parameter.
```

Reply:
I dare say @jogu is correct in his analysis here.

Reply:
also a  top level parameter in a JWT is a claim so this is a place where claim would be the right term vs. parameter 

```suggestion
`transaction_data` response parameter defined in (#transaction_data) MUST be included in the Key Binding JWT as a top level claim.
```